packages/liveblocks-codemod/README.md

```diff
@@ -42,6 +42,14 @@ are using the global types).
 npx @liveblocks/codemod@latest remove-unneeded-type-params
 ```
 
+#### `remove-clientsidesuspense-thunk`
```

Comment:
Maybe `simplify-client-side-suspense-children`?

We use dashes in all other codemods, and personally, I didn't know the word "thunk" before you used it, is it well known?

Reply:
I think Dan Abramov used it extensively in early React posts and videos, that's where I got it from. But I don't mind the name you suggested, will update! 👍

Reply:
Done in 3d084f72f72070fd728227a59cc3c6669700d557.